requirements.txt

```
@@ -1,4 +1,5 @@
 pycryptodomex
 malduck
 pymisp
-mwdblib>=3.3.0
\ No newline at end of file
+mwdblib>=3.3.0
+maco
```

```suggestion
maco-model
```

Might be better to use the `maco-model` package since it only contains the Pydantic models which is the only part in use in this feature.

Module structure is the same as the full package so you won't have to update the imports 😁